cql3/query_backend.hh

```diff
@@ -81,6 +82,7 @@ public:
             tracing::trace_state_ptr tr_state, service_permit permit, db::allow_per_partition_rate_limit allow_limit, bool raw_counters = false);
     future<bool> cas(schema_ptr schema, shared_ptr<service::cas_request> request, lw_shared_ptr<query::read_command> cmd, dht::partition_range_vector partition_ranges, coordinator_query_options query_options,
             db::consistency_level cl_for_paxos, db::consistency_level cl_for_learn, lowres_clock::time_point write_timeout, lowres_clock::time_point cas_timeout, bool write = true);
+    future<std::vector<dht::token_range_endpoints>> describe_ring(const sstring& keyspace, bool include_only_local_dc = false) const;
```

Comment:
Yet another one that does not really related to sp.

Reply:
Yeah, I bet there is still a lot more like that in sp. People add stuff to where it is most convenient to add it, not where it belongs. Also, "where it belongs" has different answers based on who you ask.